Developer_readme.md

```diff
@@ -0,0 +1,237 @@
+
+# Source code layout
```

Comment:
Could you manually wrap / change formatting of this? Going left and right kinda tough:


https://user-images.githubusercontent.com/110988/146669732-4fc2e7ce-7f52-4575-8f6b-ae6c006ca19f.mp4

Reply:
Ok I edited it, see if it is okay that way?